### PR TITLE
update title to link::standalone

### PR DESCRIPTION
**Pull request by MelSumner**
Update link::standalone to have human readable title

### CODE REVIEW

**Review by jorytindall — APPROVED** [review]

File: website/docs/components/link/standalone/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Link::Standalone
+title: Link (Standalone)
 description: A link used in isolation and not as a part of surrounding body text.
```
Comment (KristinLBradley) [inline website/docs/components/link/standalone/index.md:3]: For this, I think the more human readable option would be "Standalone Link" with "Inline Link" for the other link type. Just my opinion though.

Comment (MelSumner) [reply website/docs/components/link/standalone/index.md:3]: @KristinLBradley I thought about that- but then they wouldn't be next to each other in the list, which seems more user-friendly. WDYT?

Comment (Dhaulagiri) [reply website/docs/components/link/standalone/index.md:3]: I would make the changes in the same PR. I think they'll still be grouped next to each other based on how the form controls are showing up.

Comment (MelSumner) [reply website/docs/components/link/standalone/index.md:3]: @Dhaulagiri do you want to incorporate these changes into your existing PRs and I'll close these?

Comment (jorytindall) [reply website/docs/components/link/standalone/index.md:3]: My two cents: I read this is <name> <type>, so Link (standalone) makes cents (😉) to me. Categorically it's a link with a variant

Comment (KristinLBradley) [reply website/docs/components/link/standalone/index.md:3]: They would still be grouped under the "Link" category I would assume.

**Comment by Dhaulagiri** [comment]
Will do in https://github.com/hashicorp/design-system/pull/934

**Pull request closed without merging**